### PR TITLE
feat(rest-api-client): add get and update methods for admin notes

### PR DESCRIPTION
**Pull request by shabaraba**
## Why

- Support app.getAdminNotes method to be able to get a app admin notes.
- Support app.updateAdminNotes method to be able to update the app admin notes.

## What

- Add new methods, app.getAdminNotes and app.updateAdminNotes
- Add unit test
  - When I run `pnpm fix`, I faced the warning as below in AppClientTest:
    ```Arrow function has too many statements (41). Maximum allowed is 40  max-statements```
    So I implemented the new unit tests separately from the existing root description method.
- Add demo script
- Add type for AdminNotes

## How to test

Check test spec.

## Checklist

- [x] Read [CONTRIBUTING.md](https://github.com/kintone/js-sdk/blob/main/CONTRIBUTING.md)
- [x] Updated documentation if it is required.
- [x] Added tests if it is required.
- [x] Passed `pnpm lint` and `pnpm test` on the root directory.

### CODE REVIEW

File: packages/rest-api-client/src/client/types/app/adminNotes.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import type { AppID, Revision } from "..";
+
+export type AdminNotes = {
+  content: string;
+  includeInTemplateAndDuplicates: boolean;
+};
+
+export type AdminNoteForParameter = Partial<AdminNotes> & {
```
Comment (tasshi-me) [inline packages/rest-api-client/src/client/types/app/adminNotes.ts:8]: This type is better to be pluralized.

```suggestion
export type AdminNotesForParameter = Partial<AdminNotes> & {
```

File: packages/rest-api-client/src/client/__tests__/AppClient.test.ts [hunk 2]
```diff
@@ -1312,6 +1312,73 @@ describe("AppClient", () => {
   });
 });
 
+describe("AppClient: AdminNodes", () => {
```
Comment (tasshi-me) [inline packages/rest-api-client/src/client/__tests__/AppClient.test.ts:1315]: Seems like a typo 👍 

```suggestion
describe("AppClient: AdminNotes", () => {
```

**Review by tasshi-me — CHANGES_REQUESTED** [review]
I added some nitpick reviews.

In addition, can I ask you to rename the PR title to follow the conventional commit?

e.g. `feat(rest-api-client): add get and update methods for admin notes`

File: packages/rest-api-client/src/client/types/app/adminNotes.ts (hunk 1 above)
Comment (shabaraba) [reply packages/rest-api-client/src/client/types/app/adminNotes.ts]: [9a91117](https://github.com/kintone/js-sdk/pull/2849/commits/9a91117e943858effca9038f91ef46d61e541efd)

File: packages/rest-api-client/src/client/__tests__/AppClient.test.ts (hunk 2 above)
Comment (shabaraba) [reply packages/rest-api-client/src/client/__tests__/AppClient.test.ts]: [feb0ff1](https://github.com/kintone/js-sdk/pull/2849/commits/feb0ff14317bbcd7a03cb7bc7a1e99a535f120f7)

**Review by tasshi-me — APPROVED** [review]

**Pull request merged**